prow/cmd/checkconfig/main.go

```diff
@@ -459,7 +459,8 @@ func strictBranchesConfig(c config.ProwConfig) (*orgRepoConfig, error) {
 				for branchName := range repo.Branches {
 					branch, err := repo.GetBranch(branchName)
 					if err != nil {
-						return nil, err
+						return nil, fmt.Errorf("failed to get branch protection config for branch %s/%s/%s: %w",
```

Comment:
GetBranch doesn't seem to return an error because it couldn't get, but because what it got was bad.
Consider making this more vague (ex. "error for branch %s/%s/%s: %w") and let the wrapped error mention why.

Reply:
That was my first try too: `repo.GetBranch(branchName)` returns the error with `org/repo/branch`.
The `repo` object does not hold `org/repo` (branchName is the arg).
So I moved all 3 fields into the wrapper.

Reply:
If I understand correctly, Sean is not requesting that you change where the parameters are used, but rather the wording of this message.
`s/failed to get branch protection config/error/`

Reply:
I see. Thanks.
Updated.